scripts/genpinctrl/stm32-pinctrl-config.yaml

```diff
@@ -48,6 +48,95 @@
   match: "^DAC(?:\\d+)?_OUT\\d+$"
   mode: analog
 
+- name: ETH_COL
+  match: "^ETH_COL$"
+  mode: alternate
+
+- name: ETH_CRS
+  match: "^ETH_CRS$"
+  mode: alternate
+
+- name: ETH_CRS_DV
+  match: "^ETH_CRS_DV$"
+  mode: alternate
+
+- name: ETH_MDC
+  match: "^ETH_MDC$"
+  mode: alternate
+  slew-rate: high-speed
+
+- name: ETH_MDIO
+  match: "^ETH_MDIO$"
+  mode: alternate
+  slew-rate: high-speed
+
+- name: ETH_PPS_OUT
+  match: "^ETH_PPS_OUT$"
+  mode: alternate
+  slew-rate: high-speed
+
+- name: ETH_REF_CLK
+  match: "^ETH_REF_CLK$"
+  mode: alternate
+  slew-rate: high-speed
+
+- name: ETH_RX_CLK
+  match: "^ETH_RX_CLK$"
+  mode: alternate
+
+- name: ETH_RX_DV
+  match: "^ETH_RX_DV$"
+  mode: alternate
+
+- name: ETH_RX_ER
+  match: "^ETH_RX_ER$"
+  mode: alternate
+
+- name: ETH_RXD0
+  match: "^ETH_RXD0$"
+  mode: alternate
+
+- name: ETH_RXD1
+  match: "^ETH_RXD1$"
+  mode: alternate
+
+- name: ETH_RXD2
+  match: "^ETH_RXD2$"
+  mode: alternate
+
+- name: ETH_RXD3
+  match: "^ETH_RXD3$"
+  mode: alternate
+
+- name: ETH_TX_CLK
+  match: "^ETH_TX_CLK$"
+  mode: alternate
+
+- name: ETH_TX_EN
+  match: "^ETH_TX_EN$"
+  mode: alternate
+  slew-rate: high-speed
+
+- name: ETH_TXD0
+  match: "^ETH_TXD0$"
+  mode: alternate
+  slew-rate: high-speed
+
+- name: ETH_TXD1
+  match: "^ETH_TXD1$"
+  mode: alternate
+  slew-rate: high-speed
+
+- name: ETH_TXD2
+  match: "^ETH_TXD2$"
+  mode: alternate
+  slew-rate: high-speed
+
+- name: ETH_TXD3
+  match: "^ETH_TXD3$"
+  mode: alternate
+  slew-rate: high-speed
+
```

Comment:
In current version of the pinmux setting, all of these are `STM32_OSPEEDR_VERY_HIGH_SPEED`.
I'm not opposed ti fine tune this, but I'd prefer that we do first the transition "iso-config", so it can be seen that dts pinctrl has no functional impact.
And in a second step, we'll be able to come back on these and fin tune.